sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2299,6 +2291,13 @@ object GpuOverrides {
         override def convertToGpu(child: Expression): GpuExpression =
           GpuMakeDecimal(child, a.precision, a.scale, a.nullOnOverflow)
       }),
+    expr[Explode](
+      "Given an input array produces a sequence of rows for each value in the array",
```

Comment:
Here would be a good place to add a note about outer not being supported. It might even be nice to have something for the ExprMeta that we produce that could some how tell GenerateExec if it supports outer/inner/etc, but I am not sure if it is worth the engineering effort to do it.

Reply:
I created `GeneratorExprMeta` to wrap `outerSupport` attribute for Generators.